telethon/tl/custom/inlinebuilder.py

```diff
@@ -226,7 +235,7 @@ async def photo(
     # noinspection PyIncorrectDocstring
     async def document(
             self, file, title=None, *, description=None, type=None,
-            mime_type=None, attributes=None, force_document=False,
+            mime_type=None, attributes=None, force_document=True,
```

Comment:
Why?

Reply:
Document usually sounds for File.
And I believe, it could be breaking change for people using it, without defining parameter. Since, it was always True, before some commits.

Reply:
Changing it now would be the breaking change.